### PR TITLE
Added Some Flowers

### PR DESCRIPTION
**Pull request by matcracker**
Added Blue Orchid, Allium, Azure Bluet, Red Tulip, Orange Tulip, White
Tulip, Pink Tulip, Oxeye Daisy

### CODE REVIEW

**Comment by PEMapModder** [comment]
Flowable? Are you sure?

**Comment by matcracker** [comment]
The problem is that I do not know the ID's of the flowers ... The wiki will be wrong , I think.

File: src/pocketmine/block/Flowers.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ *
+*/
+
+namespace pocketmine\block;
+
+use pocketmine\item\Item;
+use pocketmine\level\Level;
+use pocketmine\Player;
+
+class Flowers extends Flowable{
```
Comment (PEMapModder) [inline src/pocketmine/block/Flowers.php]: Why `Flowers` not `Flower`?

Comment (matcracker) [inline src/pocketmine/block/Flowers.php]: Sorry , because I was thinking that they were more flowers instead of one xD

Comment (PEMapModder) [inline src/pocketmine/block/Flowers.php]: Do you see `Wool` being called as `Wools`? :stuck_out_tongue_winking_eye:

Comment (Falkirks) [inline src/pocketmine/block/Flowers.php]: @matcracker a `Flower` is a single flower that occupies a single block, whether it is rendered as a bunch by the client isn't relevant.

**Comment by matcracker** [comment]
I think, to have found the errors >.<

**Comment by matcracker** [comment]
I give up...  I do not know how to do, is definitely stupid mistake ç__ç

**Comment by matcracker** [comment]
Finally! No errors xD

**Comment by PEMapModder** [comment]
Rebase the 10 commits into one.

**Comment by matcracker** [comment]
Therefore I close this? And create another?

**Pull request closed without merging**